src/libexpr/eval-gc.cc

```diff
@@ -86,7 +86,8 @@ void fixupBoehmStackPointer(void ** sp_ptr, void * _pthread_id)
     auto pthread_id = reinterpret_cast<pthread_t>(_pthread_id);
     pthread_attr_t pattr;
     size_t osStackSize;
-    void * osStackLow;
+    // The low address of the stack, which grows down.
+    void * osStackLimit;
     void * osStackBase;
 
 #  ifdef __APPLE__
```

Comment:
Please change `osStackLow` on line 95.

Reply:
Ohh. I used clangd's rename, but that doesn't see through the preprocessor-disabled code.
Was nice while it lasted.